docs/SettingUp.md

```diff
@@ -45,11 +45,4 @@ If you plan to use Intellij IDEA (highly recommended):
 
 1. **Learn the design**
 
-   When you are ready to start coding, we recommend that you get some sense of the overall design by reading about [AddressBook’s architecture](DeveloperGuide.md#architecture).
-
-1. **Do the tutorials**
-   These tutorials will help you get acquainted with the codebase.
-
-   * [Tracing code](tutorials/TracingCode.md)
-   * [Adding a new command](tutorials/AddRemark.md)
-   * [Removing fields](tutorials/RemovingFields.md)
+   When you are ready to start coding, we recommend that you get some sense of the overall design by reading about [DoConnekPro's architecture](DeveloperGuide.md#architecture).
```

Comment:
The CI is not passing due to the absence of a newline at the end of this file. Just add a newline here and it should be ready to merge!